src/pages/ResearcherProfile.js

```diff
@@ -350,31 +169,56 @@ export default function ResearcherProfile(props) {
           type: 'text',
           disabled: true,
```

Comment:
Should this institution input always be set to `disabled: true`?

Reply:
yep - this is in the case that the user is a signing official and has already been assigned an institution. in this case, a user can never change their institution.